### PR TITLE
Feature/range numeric filter

### PR DESCRIPTION
**Pull request by vik753**

### CODE REVIEW

File: app/src/demo/tables/filteredTable/filters.tsx [hunk 1]
```diff
@@ -95,6 +95,20 @@ export const getFilters = (): TableFiltersConfig<Person>[] => {
...
+            columnKey: "salary",
+            title: "Salary",
+            type: "numeric",
+            predicates: [
+                { predicate: 'eq', name: '=', isDefault: true },
+                { predicate: 'neq', name: '≠'},
+                { predicate: 'lte', name: '≤'},
+                { predicate: 'gte', name: '≥'},
+                { predicate: 'inRange', name: 'In Range' },
+                // { predicate: 'notInRange', name: 'Not in Range' },
```
Comment (AlekseyManetov) [inline app/src/demo/tables/filteredTable/filters.tsx:109]: remove coment

File: epam-promo/components/filters/FiltersToolbarItem.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ const FiltersToolbarItemImpl = (props: FiltersToolbarItemProps) => {
 
     const [isOpen, isOpenChange] = useState(props.autoFocus);
     const [predicate, setPredicate] = useState(getDefaultPredicate());
-    const predicateName: string | null = predicate && props.value ? props.predicates.find(p => p.predicate === predicate).name : null;
+    const predicateName: string = props.predicates.find(p => p.predicate === predicate).name;
```
Comment (AlekseyManetov) [inline epam-promo/components/filters/FiltersToolbarItem.tsx:33]: why we need to do this if we don't have predicate and filter value?

File: epam-promo/components/filters/FiltersToolbarItem.tsx [hunk 3]
```diff
@@ -75,7 +75,7 @@ const FiltersToolbarItemImpl = (props: FiltersToolbarItemProps) => {
         <DropdownContainer>
             <Panel background="white">
                 { renderHeader() }
-                { <FilterItemBody { ...props } { ...dropdownProps } value={ getValue() } onValueChange={ onValueChange }/> }
+                { <FilterItemBody { ...props } { ...dropdownProps } currentPredicate={ { predicate, name: predicateName } } value={ getValue() } onValueChange={ onValueChange }/> }
```
Comment (AlekseyManetov) [inline epam-promo/components/filters/FiltersToolbarItem.tsx:78]: better rename currentPredicate to selectedPredicate

**Review by AlekseyManetov — CHANGES_REQUESTED** [review]

File: epam-promo/components/filters/FiltersToolbarItem.tsx (hunk 3 above)
Comment (vik753) [reply epam-promo/components/filters/FiltersToolbarItem.tsx]: done

**Pull request merged**